src/main/java/com/google/maps/model/AddressComponent.java

```diff
@@ -23,7 +24,10 @@
  * href="https://developers.google.com/maps/documentation/geocoding/intro">Google Maps Geocoding API
  * Developer's Guide</a> for more detail.
  */
-public class AddressComponent {
+public class AddressComponent implements Serializable {
+  /** The default serialVersionUID */
```

Comment:
I think this comment is probably redundant (on all classes).

Reply:
Thanks Sam, I concur but as this was the first time we introduce Serializable, I thought that this comment was neccessary. But I also agree with you that probably it's redundant in every class.
Should I delete them before you aprobed this pull request? 
I tried to make real notice that that field was introduced for the serializable interface

Reply:
@lfochi please remove, and then I'll merge. Thanks!